docs/basic-docs/atomicdex-api-20-dev/enable_bch_with_tokens.md

```diff
@@ -0,0 +1,591 @@
+```
+
+
+## Error (Unsafe configuration with empty bchd_urls param)
+```json
+{
+  "mmrpc":"2.0",
+  "error":"Error Using empty bchd_urls is unsafe for SLP users! on platform coin BCH creation",
+  "error_path":"platform_coin_with_tokens.bch_with_tokens_activation",
+  "error_trace":"platform_coin_with_tokens:290] bch_with_tokens_activation:212]",
+  "error_type":"PlatformCoinCreationError",
+  "error_data":{
+    "ticker":"BCH",
+    "error":"Using empty bchd_urls is unsafe for SLP users!"
+  },
+  "id":null
+}
+```
+
+
+## Error (conf file not found when enabling in native mode)
+```json
+{
+  "mmrpc":"2.0",
+  "error":"Error bch:633] utxo:1704] utxo:995] Error parsing the native wallet configuration '/home/user/.Bitcoin Cash Testnet/Bitcoin Cash Testnet.conf': No such file or directory (os error 2) on platform coin tBCH creation",
+  "error_path":"platform_coin_with_tokens.bch_with_tokens_activation",
+  "error_trace":"platform_coin_with_tokens:290] bch_with_tokens_activation:212]",
+  "error_type":"PlatformCoinCreationError",
+  "error_data":{
+    "ticker":"tBCH",
+    "error":"bch:633] utxo:1704] utxo:995] Error parsing the native wallet configuration '/home/user/.Bitcoin Cash Testnet/Bitcoin Cash Testnet.conf': No such file or directory (os error 2)"
+  },
+  "id":null
+}
+```
+
+
+## Error (Electrum server not responding)
+```json
+{
+  "mmrpc":"2.0",
+  "error":"Error bch:633] utxo:1667] Failed to connect to at least 1 of [ElectrumRpcRequest { url: \"bch0.kister.net:5100\", protocol: TCP, disable_cert_verification: false }, ElectrumRpcRequest { url: \"testnet.imaginary.cash:5000\", protocol: TCP, disable_cert_verification: false }, ElectrumRpcRequest { url: \"blackie.c3-soft.com:6000\", protocol: TCP, disable_cert_verification: false }, ElectrumRpcRequest { url: \"tbch.loping.net:6000\", protocol: TCP, disable_cert_verification: false }, ElectrumRpcRequest { url: \"electroncash.de:5000\", protocol: TCP, disable_cert_verification: false }] in 5 seconds. on platform coin tBCH creation",
+  "error_path":"platform_coin_with_tokens.bch_with_tokens_activation",
+  "error_trace":"platform_coin_with_tokens:290] bch_with_tokens_activation:212]",
+  "error_type":"PlatformCoinCreationError",
+  "error_data":{
+    "ticker":"tBCH",
+    "error":"bch:633] utxo:1667] Failed to connect to at least 1 of [ElectrumRpcRequest { url: \"bch0.kister.net:5100\", protocol: TCP, disable_cert_verification: false }, ElectrumRpcRequest { url: \"testnet.imaginary.cash:5000\", protocol: TCP, disable_cert_verification: false }, ElectrumRpcRequest { url: \"blackie.c3-soft.com:6000\", protocol: TCP, disable_cert_verification: false }, ElectrumRpcRequest { url: \"tbch.loping.net:6000\", protocol: TCP, disable_cert_verification: false }, ElectrumRpcRequest { url: \"electroncash.de:5000\", protocol: TCP, disable_cert_verification: false }] in 5 seconds."
+  },
+  "id":null
+}
+```
+
+
+
+# enable\_slp
+
+The `enable_slp` method allows you to activate additional SLP tokens
```

Comment:
Could you please mention that this method requires the prior activation of the platform coin? And also move this to a separate file?